public/js/bio-link/share.js

```
@@ -0,0 +1,140 @@
+    modal.classList.add('show');
+}
+
+function closeShareBioLinkModal() {
+    const modal = document.getElementById('shareBioLinkModal');
+    if (modal) {
+        modal.style.display = 'none';
+        modal.classList.remove('show');
+    }
+}
+
+function copyShareBioLink() {
```

ESLint

[error] 'copyShareBioLink' is defined but never used.

(no-unused-vars)